### ISSUE TITLE
BetterFps + Optifine will make me go all over the place in minecraft

### ISSUE THREAD

**Issue by SirLagsAWholeLot** [Guichaguri/BetterFps#12]
Hi. I am using betterfps installed onto the optifine instance for minecraft 1.10, and it seemed to be working, but whenever I went into my game my arms would wildly flail around in third person and whenever I move, my camera goes insane. I'm not using forge with this, and yes I've tried using forge but the problem still persisted. Also, the arm flailing looks like the steves from really early minecraft. Please help.
![2016-07-10_18 31 28](https://cloud.githubusercontent.com/assets/20387969/16716799/a701c41a-46cc-11e6-9933-34c74f55075c.png)
![2016-07-10_18 31 21](https://cloud.githubusercontent.com/assets/20387969/16716800/ab2a566a-46cc-11e6-877c-443aca5edb41.png)
![2016-07-10_18 31 27](https://cloud.githubusercontent.com/assets/20387969/16716810/cfae7eee-46cc-11e6-88f4-ec4d6a5d64dd.png)

**Comment by Guichaguri** [comment]
Which algorithm are you using?

**Comment by SirLagsAWholeLot** [comment]
What do you mean? :P

**Comment by Guichaguri** [comment]
Press F12 in-game, it will open a GUI with an algorithm selector.
Choose another algorithm, it might fix the problem.
Just let me know which algorithm is causing the problem :)

**Comment by Guichaguri** [comment]
I'll be closing this issue for inactivity. Feel free to reopen it if the problem still persist.

**Comment by jakeanq** [comment]
Hello, I have noticed a very similar issue with Taylor's algorithm, however I am using Minecraft 1.11.2 with no Optifine (quite a lot of other mods though).  I noticed differing movement speeds, arms and legs spinning wildly (both on player and on mobs) and rendering glitches on torch flames.  No issues are noticed on Riven's half algorithm.

Graphics: NVidia GeForce GTX 765M
OS: Arch Linux x86_64

![2017-07-10_20 50 51](https://user-images.githubusercontent.com/3282717/28064122-fa1a3e54-6622-11e7-974f-5332546f4b83.png)
![2017-07-10_21 02 47](https://user-images.githubusercontent.com/3282717/28064133-045d225a-6623-11e7-8bf8-f2c5adc4b7d6.png)
